test/finders/songs_finder_test.rb

```diff
@@ -3,7 +3,9 @@
 class SongsFinderTest < ActionController::TestCase
   single_word_results = SongsFinder.search "relevant"
   multi_word_results = SongsFinder.search "truth live life hands"
-  
+  partial_word_results = SongsFinder.search 'hand'
+  all_partial_word_results_2 = SongsFinder.search 'go b prais io'
```

Comment:
this looks like a one off variable/list, so I'd just use it in the test down below